index.js

```diff
@@ -35,7 +35,11 @@ var Orng = function(serial, callback) {
         .then(function(properties) {
           var model = properties['ro.product.model'];
 
-          orng.devicePixelRatio = properties['ro.sf.lcd_density'] / 160;
+          if (model.indexOf('Emulator') != -1) {
+            orng.devicePixelRatio = properties['qemu.sf.lcd_density'] / 160;
```

Comment:
I'm wondering if instead of doing a conditional like this here, if we should add an overridable config option like this:

In config.json:

```
"Full Android On Emulator": {
  "densityProperty": "qemu.sf.lcd_density"
}
```

Then in index.js:

```js
var densityProperty = config.densityProperty || 'ro.sf.lcd_density';

orng.devicePixelRatio = properties[densityProperty] / 160;
```

Thoughts?